docs/configuration/coredns-configuration.md

```diff
@@ -0,0 +1,141 @@
+# Configuration of CoreDNS in microk8s to use different nameservers for different domains and ip ranges
+
+
+In MicroK8s, CoreDNS is enabled by running the following command: `microk8s enable dns`.
+
+Alternatively, you can specify a list of DNS servers by running the command: `microk8s enable dns:8.8.8.8,1.1.1.1`.
+
+The servers in the provided list are expected to be capable of resolving the same addresses. 
+If one of these servers is unreachable, another one is used. 
+If the requirement is to use different DNS servers for various domains or different IP ranges in the case of reverse lookup, the configuration differs.
+
+Before executing `microk8s enable dns`, the first step is to edit `coredns.yaml`, located inside the MicroK8s installation folder.
+An example path is: `/var/snap/microk8s/common/addons/core/addons/dns/coredns.yaml`.
+
+
+Inside `coredns.yaml`, there is a complete configuration for the CoreDNS deployment.
+The only section that requires editing is the ConfigMap:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: coredns
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: EnsureExists
+    k8s-app: kube-dns
+data:
+  Corefile: |
+    .:53 {
+        errors
+        health {
+          lameduck 5s
+        }
+        ready
+        log . {
+          class error
+        }
+        kubernetes cluster.local in-addr.arpa ip6.arpa {
+          pods insecure
+          fallthrough in-addr.arpa ip6.arpa
+        }
+        prometheus :9153
+        forward . $NAMESERVERS
+        cache 30
+        loop
+        reload
+        loadbalance
+    }
+```
+
+Changes should be made in `data.Corefile` within this ConfigMap. This documentation explains basic configuration. 
```

Comment:
Maybe to not repeat `this`, we can change one of `this` to `presented`, ex. Presented documentation ...